src/lib/events/events.ts

```diff
@@ -66,7 +66,7 @@ export class Events {
         return data
     }
 
-    async update(eventId: number, event: TimelyEvent): Promise<TimelyEvent> {
+    async update(eventId: number, event: Partial<TimelyEvent>): Promise<TimelyEvent> {
         const { data } = await this.http.put(`/${this.config.accountId}/events/${eventId}`, event)
```

Comment:
Does Timely's API behave like a `PATCH` even on a  `PUT` request?

Reply:
Yeah, it's weird, even their documentation shows the puts as a patch. https://dev.timelyapp.com/#update-a-project